.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "examples/pipelines/sklearn_regression"]
-	path = examples/pipelines/sklearn_regression
-	url = https://github.com/mlflow/mlp-regression-template.git
+[submodule "examples/pipelines/sklearn_regression_example"]
+	path = examples/pipelines/sklearn_regression_example
```

Comment:
We probably have to rename the directory later, but let's leave it as `pipelines` for now.